Makefile

```diff
@@ -6,16 +6,31 @@ BUILD_IMAGE_VERSIONS = $(BUILD_IMAGE)_3.0 $(BUILD_IMAGE)_2.6 $(BUILD_IMAGE)_2.5
 
 ${BUILD_IMAGE}: $(BUILD_IMAGE_VERSIONS)
 
+# BUILDX_OUTPUT defines the buildx output
+# --load builds locally the container image
+# --push builds and pushes the container image to a registry
+BUILDX_OUTPUT ?= --push
+
+# Supported platforms for the builder image
+PLATFORMS ?= linux/arm64,linux/amd64
+
+TARGET_OS ?= linux
+
+# BUILDX_BUILD_ARGS are the additional --build-arg flags passed to the docker buildx build command.
+BUILDX_BUILD_ARGS = --build-arg TARGETOS=$(TARGET_OS)
+
 # Build a specific maistra image. Example of usage: make maistra-builder_2.3
 ${BUILD_IMAGE}_%:
 	$(CONTAINER_CLI) build -t ${HUB}/${BUILD_IMAGE}:$* \
 				 -f docker/$@.Dockerfile docker
 
-# Build a specific arm64 maistra image. Example of usage: make maistra-builder_2.4_arm64
-${BUILD_IMAGE}_%_arm64:
-	$(CONTAINER_CLI) build -t ${HUB}/${BUILD_IMAGE}:$* \
-				 --build-arg TARGETARCH=arm64 \
-				 -f docker/$@.Dockerfile docker
+# Build a specific arm64 maistra image. Example of usage: make maistra-builder_2.5_multi
+# This target works from 2.5 version
+${BUILD_IMAGE}_%_multi:
+	- docker buildx create --name project-v4-builder
```

Comment:
Should we replace docker with "$(CONTAINER_CLI)" ? 
Is this buildx only working with docker  and not working with podman ?

Reply:
Done, and I added the conditions to use also podman to build the multi image. Testing now this (It will take a while to finish the build)